plugins/content/caption/search/CaptionSearchPlugin.php

```diff
@@ -255,15 +255,16 @@ public static function getCaptionElasticSearchData($entry)
 				continue;
 
 			$language = $captionAsset->getLanguage();
-			self::getElasticLines($captionData, $captionRawData, $items, $language, $captionAsset->getId(), $captionAsset->getLabel());
+
+			self::getElasticLines($captionData, $captionRawData, $items, $language, $captionAsset->getId(), $captionAsset->getUsage(), $captionAsset->getLabel(), $captionAsset->getAccuracy());
```

Comment:
in the above function we choose the highest accuracy in 
filterCaptionsByLangAndLabel
that means
for each key 
			$key = "{$captionLanguage}_{$captionLabel}";
we save the asset with highest accuracy

that means we **save the highest accuracy to elastic**, maybe that is enough for our usecase, but this means that our filters are not generic 
e.g. 
accuracyLessThan 
should be 
maxAccuracyLessThan OR highestAccuracyLessThan


due to the same function, we do not consider usage 
meaning 
if we have EAD and captions with the same captionLanguage and captionLabel 
they will override one-another